partner_risk_insurance/models/partner.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 AvancOSC - Daniel Campos
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.depends('company_credit_limit', 'insurance_credit_limit')
+    def _compute_credit_limit(self):
+        for partner in self:
+            partner.credit_limit = (partner.company_credit_limit +
+                                    partner.insurance_credit_limit)
+
+    credit_limit = fields.Float(string='Credit Limit',
```

Comment:
You shouldn't make this field computed. Please make as in v8, writing this field on write.